packages/expo-gl-cpp-legacy/cpp/EXGLContext.cpp

```diff
@@ -2,7 +2,10 @@
 
 #include <JavaScriptCore/JSContextRef.h>
 
+#ifndef APPLE
```

Comment:
There is error here it should be `#ifndef __APPLE__`

according to stb docs `STB_IMAGE_IMPLEMENTATION` should be defined in one cpp file (so symbols are included only once). In this case STB_IMAGE_IMPLEMENTATION is already defined in expo-gl-cpp (non legacy).

It's only done on iOS because symbol collision does not happen on Android, I'm not 100% sure why, but it's either because internal symbols are mangled or prefixed so two libraries symbols won't collide with each other

Reply:
Hm, makes sense — do you think adding this information to the comment makes sense?

Reply:
there is a comment there that it's already included in expo-gl-cpp, probably could be better, but doesn't really matter, expo-gl-cpp-legacy won't be developed any further